### ISSUE TITLE
"Error: request aborted" on >= v0.3.0

### ISSUE THREAD

**Issue by kellyselden** [kategengler/ember-cli-code-coverage#39]
I've been toggling between v0.2.2 and v0.3.0 (or v0.3.1) and I'm getting a cryptic error in the new one.

```
1..1163
# tests 1163
# pass  1141
# skip  22
# fail  0

# ok
Error: request aborted
    at IncomingMessage.onAborted (/my-project-path/node_modules/raw-body/index.js:269:10)
    at emitNone (events.js:86:13)
    at IncomingMessage.emit (events.js:185:7)
    at abortIncoming (_http_server.js:280:11)
    at Socket.serverSocketCloseListener (_http_server.js:293:5)
    at emitOne (events.js:101:20)
    at Socket.emit (events.js:188:7)
    at TCP._handle.close [as _onclose] (net.js:493:12)
```

And it doesn't create the coverage folder. It worked fine in the older version. I'm not using mirage or pretender.

**Comment by rwjblue** [comment]
Using phantom?  Mocha? Sounds conceptually similar to https://github.com/kategengler/ember-cli-code-coverage/issues/32 (in that it sounds related to reading the posted coverage payload).

**Comment by kellyselden** [comment]
phantom and qunit

**Comment by tylergets** [comment]
Having the same issue.

**Comment by denchen** [comment]
I'm seeing the same issue too. I've tried `0.3.1` and `0.3.2`, and I get that `request aborted` message, and there is no `/coverage/` directory. I am using **`ember-cli-mirage@0.2.2`** with this passthrough in `/mirage/config.js`:

      this.passthrough('/write-coverage');

I went down to `0.2.2` and everything seems to work, and I get coverage information.

**Comment by nathanpalmer** [comment]
We're running into something similar. Getting timeouts on 0.3.2. Downgraded to 0.2.2 and it's working fine. We also use ember-cli-mirage@(roughly 0.2.2).

**Comment by JoshSmith** [comment]
Also having the same issue, using Mirage with the passthrough.

**Comment by kategengler** [comment]
#57 seems similar

**Comment by baseballlover723** [comment]
having the same issue on one of my projects using 0.3.2. Whats weird is that it only sometimes (~50%) happens and only on codeship. I can't reproduce this locally. Downgrading to 0.2.2 seems to of fixed it

**Comment by Senthe** [comment]
There was similar issue for Blanket: 
https://github.com/sglanzer/ember-cli-blanket/issues/112. It seems that making the request to /write-coverage synchronous might help. Now if I only knew what to change in my code to try it out.

**Comment by kategengler** [comment]
The request did change from being sync to async in this commit https://github.com/kategengler/ember-cli-code-coverage/commit/f2ba0e8091daedc20e4d196138fbf9f841f8eb3f#diff-94eab975fb48e219fc130d82e21457ac

However, I have a hunch that this is related to testem version and the fact that a synchronous request fixes it is a patch over the problem, rather than the solution. 


**For anybody seeing this issue, could you please provide:**

- Your ember-cli version
- Whether you're running `ember test` `ember test -s` or navigating to `/tests`

**Comment by chrisvdp** [comment]
`"ember-cli": "^2.8.0"`
running `COVERAGE=true $(npm bin)/ember test` in CI

**Comment by Cryrivers** [comment]
we are running into the same issue. 

Ember CLI: 2.8.0
Running `COVERAGE=true ember test`

However, running `COVERAGE=true ember test --server` works.

**Comment by adhamhf** [comment]
"ember-cli": "2.8.0",
  "ember-cli-code-coverage": "^0.3.6",
  `COVERAGE=true ember test --launch phantomjs`  fails
  `COVERAGE=true ember test --server` works

**Comment by andrew-zenefits** [comment]
"ember-cli": 2.4.3,
"ember-cli-code-coverage": "^0.3.8",

`COVERAGE=true ember t` fails 
`COVERAGE=true ember t -s` works (meaning it will update the `/coverage/*` files

**Comment by ethanrubio** [comment]
Same issue here on circleci. It works locally, however. 

"ember-cli": "2.10.0",
"ember-cli-code-coverage": "^0.3.8",

`COVERAGE=true ember test` fails

**Comment by wagenet** [comment]
Appears to be fixed by #82

**Comment by kellyselden** [comment]
Just tried it out. Works for me!
